src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1277,8 +1277,10 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 }
             }
 
-            // Add the content headers
-            if (request.Content == null)
+            // Add the content headers except for Get Method
```

Comment:
Can you please reference the RFC
```suggestion
            // Add the content headers except for Get Method per RFC7230
```